examples/DocExampleApis/Program.cs

```diff
@@ -122,4 +137,134 @@ public static async Task Example_API_Delete(CacheClient cacheClient)
             throw new Exception($"An error occurred while attempting to delete key 'test-key' from cache 'test-cache': {error.ErrorCode}: {error}");
         }
     }
+
+    public static async Task Example_API_GenerateDisposableToken(IAuthClient authClient)
+    {
+        // Generate a disposable token with read-write access to a specific key in one cache
+        var oneKeyOneCacheToken = await authClient.GenerateDisposableTokenAsync(
+            DisposableTokenScopes.CacheKeyReadWrite("squirrels", "mo"),
+            ExpiresIn.Minutes(30)
+        );
+        if (oneKeyOneCacheToken is GenerateDisposableTokenResponse.Success token1)
+        {
+            

```

Comment:
I don't normally comment about whitespace, but looking good is half the point of examples :-).